src/protocol/resp/src/request/mod.rs

```diff
@@ -87,6 +89,9 @@ impl Parse<Request> for RequestParser {
 
                 match &array[0] {
                     Message::BulkString(c) => match c.inner.as_ref().map(|v| v.as_ref().as_ref()) {
+                        Some(b"badd") | Some(b"BADD") => {
```

Comment:
In the future we may consider using conditional compilation to enable/disable groups of commands. For now this is probably okay as-is.